input/resources/au-core-diagnosticresult-path.xml

```diff
@@ -141,8 +141,8 @@
         <key value="au-core-obs-01"/>
         <severity value="error"/>
         <human value="Date shall be at least to day"/>
```

Comment:
Suggest updating to reflect addition of DAR extension eg "Date shall be at least to day or, if not available, the Data Absent Reason extension shall be present"